sentry_sdk/integrations/celery.py

```diff
@@ -427,17 +440,17 @@ def _get_humanized_interval(seconds):
     for unit, divider in TIME_UNITS:
         if seconds >= divider:
             interval = int(seconds / divider)
-            return (interval, unit)
+            return (interval, cast("MonitorConfigScheduleUnit", unit))
```

Comment:
mypy is not smart enough that `unit` comes from `TIME_UNITS` which is an immutable tuple of tuples where the first element is a subset of `MonitorConfigScheduleUnit`. Trying to add a type comment to line 437 to tell mypy the precise type of `TIME_UNITS` doesn't work either. So we cast.

Reply:
@szokeasaurusrex Do you think this `cast` is okay like this (aliased to `lambda _, x: x` at runtime) or should the `cast` itself be hidden behind `TYPE_CHECKING` here?

Reply:
`cast` probably needs to be behind `TYPE_CHECKING` to maintain compatibility with Python 2.x.

Reply:
Nvm, looks like you handled the compatibility concern by importing as `lambda _, x: x` when the import fails. Although, worth noting, that I think this `lambda` only gets activated on Python 2.x. In runtime on Python 3.5+, we still call `cast` from `typing`, but that is fine because it is a no-op that returns the second argument at runtime.